highfidelity/hifi
# WL 21551 - Flocking Fish Aquarium

> replace the basic block with a nice looking model of a fish. 
> And when the script is run, also create a simple translucent walled 'aquarium' that the fish stay inside. 
> use a tablet button to make/delete the acquarium and fish. 
> put the results in the marketplace

Worklist [21551](https://worklist.net/21551)

## Test Plan
1. Run the FlockOfFish.js script from here: [FlockOfFish.js](https://s3-eu-west-1.amazonaws.com/cainkilgore-prod/files/FlockOfFish.js
)
2. Confirm that a "FishTank" Toolbar/Tablet Button gets added
3. Confirm that clicking on it toggles the state off and on and a fishtank appears in front of you.
4. Make sure the fishes spawned in correctly and move around.

Marketplace Item: [here (not available yet)](https://highfidelity.com/marketplace/items/4535b57c-f35a-4d9d-a3ee-1233c457dc8e
)

![5e209ad7-8628-4c9e-bf5b-1745e676fbd0](https://user-images.githubusercontent.com/898687/30995525-c1bca24e-a4b2-11e7-839e-52258282fb64.gif)
